gef-ui/server/controllers/about-exporter/index.js

```diff
@@ -78,12 +78,12 @@ const validateAboutExporter = async (req, res) => {
       }
     }
 
-    // Regex tests to see if value is a number between 0 and 100
-    const zeroToOneHundredRegex = /^([0-9]|[1-9][0-9]|100)$/;
-    if (body.probabilityOfDefault && !zeroToOneHundredRegex.test(body.probabilityOfDefault)) {
+    const maxPercentage = 14.1;
+    const probabilityOfDefault = Number(body.probabilityOfDefault);
+    if (Number.isNaN(probabilityOfDefault) || probabilityOfDefault > maxPercentage) {
```

Comment:
Not one for the first day but extract this to a utils method of isNumberGreaterThan(maxPercentage) or the like

Reply:
This is a specific use case for validation so doesn't really suit being a util. Returns true if it is not a number for example.